src/main/java/com/ericsson/eiffelcommons/jenkins/JenkinsManager.java

```diff
@@ -165,18 +153,21 @@ public boolean buildJob(String jobName, String jobToken) throws Exception {
     /**
      * This function that trigger a build on a jenkins job with given body
      *
-     * @param jobName
-     *            :: Name of job as String
-     * @param jobToken
-     *            :: Token used to trigger the job as String
-     * @param body
-     *            :: Body as string "json={param:"param"}"
+     * @param jobName  :: Name of job as String
+     * @param jobToken :: Token used to trigger the job as String
+     * @param body     :: Body as string "json={param:"param"}"
      * @return
      * @throws Exception
      */
-    public boolean buildJobWithFormPostParams(String jobName, String jobToken, String body) throws Exception {
+    @Deprecated
+    public boolean buildJobWithFormPostParams(String jobName, String jobToken, String body)
+            throws Exception {
+        // ! This method currently doesn't work with a fixed GET request as it will fail to set the
```

Comment:
This is possible if the job is configured correctly and body must start with "json={your parameters}"
Update the javadoc with this information please.

Reply:
The error I get is:
java.lang.ClassCastException: org.apache.http.client.methods.HttpGet cannot be cast to org.apache.http.client.methods.HttpEntityEnclosingRequestBase
It happens when it tries to cast HttpGet to set the body.

Reply:
I looked more into it and only HttpPatch, HttpPost and HttpPut are subclasses of HttpEntityEnclosingRequestBase.

Reply:
You basically have to do a special solution such as the one seen here: https://stackoverflow.com/questions/12535016/apache-httpclient-get-with-body
to attach a body to a GET request.

Reply:
Hmm it should be a POST and not GET. Read more about this triggering here https://wiki.jenkins.io/display/JENKINS/Remote+access+API

Reply:
If it's supposed to be a POST request then I can make the changes to make this work.